README.md

```diff
@@ -14,7 +14,19 @@ mkdir build
 cd build
 cmake ..
 make -j3
-./ftxuistarter
+cd ../target
+./ftxui-starter
+~~~
+
+## Webassembly build:
+~~~bash
+mkdir build_emscripten
+cd build_emscripten
+emcmake cmake ..
+make -j3
+cd ../target
```

Comment:
This line isn't useful anymore, because of my previous comment about not copying files into the "./target" directory.